src/styles/blocks/header.scss

```diff
@@ -0,0 +1,80 @@
+@import '../utils/variables';
+
+.header {
+  width: 100vw;
+  font-size: $font-size-medium;
+  font-weight: 500;
+  text-transform: uppercase;
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-right: 50px;
+    width: 100%;
+    box-shadow: 0 2px 4px 0 #0000000d;
+
+    .logo__img {
+      display: flex;
+      padding-left: 50px;
+      width: 40px;
+      height: 40px;
+    }
+
+    .nav {
+      display: flex;
+
+      &__list {
+        display: flex;
+        margin: 0;
+        padding: 0;
+        list-style: none;
+        align-items: center;
+      }
+
+      &__link.is-active {
```

Comment:
The class `is-active` is used here and also defined globally. Ensure that this class is used consistently across your styles to avoid conflicts or unexpected behavior.